tools/src/dynamic-macros/macros.ts

```diff
@@ -199,7 +175,8 @@ export default {
       return '';
     }
 
-    const url = await UrlUtils.constructManifestUrlAsync(EXPO_GO_DIR);
+    const { address } = await lanNetwork();
+    const url = `exp://${address}:${EXPO_GO_DEV_SERVER_PORT}`;
```

Comment:
xdl's `UrlUtils.constructManifestUrlAsync` will guess running dev server from a saved file in **.expo**. we didn't do that in nowadays. we use a fixed port for expo-go's home app